### DISCUSSION TITLE
posts/GCP_Authentication_Without_JSON_file/

### DISCUSSION THREAD

**Discussion by giscus** [bitesinbyte/blog#3] [Q&A] [answered]
# posts/GCP_Authentication_Without_JSON_file/

In this blog post, I will explain how we can authenticate GCP or Firebase without storing JSON in your repository.

https://blogs.bitesinbyte.com/posts/GCP_Authentication_Without_JSON_file/

**Comment by manishtiwari25** [comment, marked as answer]
Nice stuff, thanks :)

**Reply by manishtiwari25** [reply]
Just a test :) It works 🥳
